libsignal-service/src/sender.rs

```diff
@@ -691,6 +693,51 @@ where
         Ok(())
     }
 
+    /// Send `MessageRequestResponse` synchronization message with either a recipient ACI or a GroupV2 ID
+    #[tracing::instrument(skip(self))]
+    pub async fn send_message_request_response(
+        &mut self,
+        recipient: &ServiceAddress,
+        thread_aci: Option<&ServiceAddress>,
+        group_id: Option<&[u8; 64]>,
```

Comment:
Don't we have a stronger type for this?

Reply:
I looked for one but didn't find one. Not for the length either.

I think this is better resolved in another PR - I'll make an issue.